### DISCUSSION TITLE
Adding a scheduler to Adonisjs

### DISCUSSION THREAD

**Discussion by benjibenoit** [adonisjs/core#4389] [Ideas]
Hi

I've seen quite a few requests on this subject and I myself need a scheduler. Would it be possible to imagine a system a bit like AgendaJS, i.e. with job persistence and recovery in the event of server shutdown?
Or do you have any suggestions for me to make one?
PS: I'm using adonisjs 6

Thanks !

**Comment by MichaelBelgium** [comment]
There is only one scheduler package i know that supports adonisv6: [click](https://github.com/KABBOUCHI/adonisjs-scheduler)
